apache/ambari
# AMBARI-25169 - Upgrade Apache Solr version to 7.7.0 or later in Ambari

## What changes were proposed in this pull request?

CVE-2017-3164 SSRF issue in Apache Solr
https://wiki.apache.org/solr/SolrSecurity

## How was this patch tested?

Manually:
1. Build infra-solr
2. Deploy ambari and a cluster including zookeeper, infra-solr, logsearch
3. Replace infra-solr instance with the new build
4. Check logsearch UI for logs

cc: @g-boros
